### PR TITLE
Mods amb infos

### PR DESCRIPTION
**Pull request by BastienSaulnier**

### CODE REVIEW

**Review by LeoDiduch — DISMISSED** [review]
SnE

File: frontend/src/pages/AmbassadorInfos.jsx [hunk 1]
```diff
@@ -1,53 +1,99 @@
...
         </div>
 
-        <p>{ambassador.resume}</p>
+        <div className="content">
+          <div className="tags">
+            <ul>
+              <li>
+                <img
+                  className="profileTag"
+                  src="https://via.placeholder.com/50x50"
```
Comment (LoicBrassart) [inline frontend/src/pages/AmbassadorInfos.jsx]: No. More. Placeholders.
Ever.

File: frontend/src/pages/AmbassadorInfos.jsx [hunk 2]
```diff
@@ -1,53 +1,99 @@
...
+                  alt="..."
+                />
+              </li>
+              <li>
+                <img
+                  className="profileTag"
+                  src="https://via.placeholder.com/50x50"
```
Comment (LoicBrassart) [inline frontend/src/pages/AmbassadorInfos.jsx]: No. More. Placeholders.
Ever.

File: frontend/src/pages/AmbassadorInfos.jsx [hunk 3]
```diff
@@ -1,53 +1,99 @@
...
```
Comment (LoicBrassart) [inline frontend/src/pages/AmbassadorInfos.jsx]: No. More. Placeholders.
Ever.

File: frontend/src/pages/CauseInfos.jsx [hunk 4]
```diff
@@ -1,54 +1,102 @@
...
+            mainStyle="ProfileCardYellow"
+          />
         </div>
+        <div className="content">
+          <div className="tags">
+            <ul>
+              <li>
+                <img
+                  className="profileTag"
+                  src="https://via.placeholder.com/50x50"
```
Comment (LoicBrassart) [inline frontend/src/pages/CauseInfos.jsx]: No. More. Placeholders.
Ever.

File: frontend/src/pages/CauseInfos.jsx [hunk 5]
```diff
@@ -1,54 +1,102 @@
...
+                  alt="..."
+                />
+              </li>
+              <li>
+                <img
+                  className="profileTag"
+                  src="https://via.placeholder.com/50x50"
```
Comment (LoicBrassart) [inline frontend/src/pages/CauseInfos.jsx]: No. More. Placeholders.
Ever.

File: frontend/src/pages/CauseInfos.jsx [hunk 6]
```diff
@@ -1,54 +1,102 @@
...
```
Comment (LoicBrassart) [inline frontend/src/pages/CauseInfos.jsx]: No. More. Placeholders.
Ever.

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
No. More. Placeholders.
Ever.

File: frontend/src/pages/AmbassadorInfos.jsx (hunk 1 above)
Comment (LeoDiduch) [reply frontend/src/pages/AmbassadorInfos.jsx]: need other branch to add good src

File: frontend/src/pages/AmbassadorInfos.jsx (hunk 2 above)
Comment (LeoDiduch) [reply frontend/src/pages/AmbassadorInfos.jsx]: need other branch to add good src

File: frontend/src/pages/AmbassadorInfos.jsx (hunk 3 above)
Comment (LeoDiduch) [reply frontend/src/pages/AmbassadorInfos.jsx]: need other branch to add good src

File: frontend/src/pages/CauseInfos.jsx (hunk 4 above)
Comment (LeoDiduch) [reply frontend/src/pages/CauseInfos.jsx]: need other branch to add good src

File: frontend/src/pages/CauseInfos.jsx (hunk 5 above)
Comment (LeoDiduch) [reply frontend/src/pages/CauseInfos.jsx]: need other branch to add good src

File: frontend/src/pages/CauseInfos.jsx (hunk 6 above)
Comment (LeoDiduch) [reply frontend/src/pages/CauseInfos.jsx]: need other branch to add good src

**Review by LeoDiduch — DISMISSED** [review]
SnE, placeHolder change when pull request with backend routes is completed

**Review by LeoDiduch — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**